src/managed/OpenLiveWriter.Localization/Strings.resx

```diff
@@ -4486,4 +4486,7 @@ This might take a while.</value>
   <data name="UnexpectedErrorSendError" xml:space="preserve">
     <value>&amp;Send Error</value>
   </data>
+  <data name="percent" xml:space="preserve">
```

Comment:
I'd copied the convention for 'pixels' being lowercased, but happy/relieved to fix this. I might fix 'pixels' to be the same while I'm there.